icmp.go

```diff
@@ -39,15 +41,63 @@ func getICMPSequence() uint16 {
 }
 
 func probeICMP(target string, w http.ResponseWriter, module Module) (success bool) {
+	var (
+		socket           *icmp.PacketConn
+		requestType      icmp.Type
+		replyType        icmp.Type
+		fallbackProtocol string
+	)
+
 	deadline := time.Now().Add(module.Timeout)
-	socket, err := icmp.ListenPacket("ip4:icmp", "0.0.0.0")
+
+	// Defaults to IPv4 to be compatible with older versions
```

Comment:
OK, but as this one could potentially break monitoring for users, warning into release announcement, please.